src/scf-release/src/acceptance-tests-brain/test-scripts/018_autoscaler_test.rb

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env ruby
+require_relative 'testutils'
+
+# APPS_DOMAIN   = <%= p('smoke_tests.apps_domain') %>
+# BROKER_URL    = <%= p('smoke_tests.autoscaler_service_broker_endpoint') %>
+# BROKER_USER   = <%= p('autoscaler.service_broker.username') %>
+# BROKER_PASS   = <%= p('autoscaler.service_broker.password') %>
+# SMOKE_SERVICE = <%= p('autoscaler.smoke.service_name') %>
+# SMOKE_PLAN    = <%= p('autoscaler.smoke.service_plan') %>
+
+APPS_DOMAIN   = ENV['APPS_DOMAIN']
+BROKER_USER   = ENV['BROKER_USER']
+BROKER_PASS   = ENV['BROKER_PASS']
+BROKER_URL    = ENV['BROKER_URL']
+SMOKE_SERVICE = ENV['SMOKE_SERVICE']
+SMOKE_PLAN    = ENV['SMOKE_PLAN']
+NAMESPACE     = ENV['KUBERNETES_NAMESPACE']
+
+STATEFULSETS = [ 'autoscaler-api',
+                 'autoscaler-eventgenerator',
+                 'autoscaler-metrics',
+                 'autoscaler-operator',
+                 'autoscaler-postgres',
+                 'autoscaler-scalingengine',
+                 'autoscaler-scheduler',
+                 'autoscaler-servicebroker' ]
+
+# ~ ~~ ~~~ ~~~~~ ~~~~~~~~ ~~~~~~~~~~~~~ ~~~~~~~~~~~~~~~~~~~~~
+## Check if autoscaler pods are running
+
+have = 0
+STATEFULSETS.each do |name|
+  have += 1 if statefulset_ready(NAMESPACE, name)
+end
+
+# Skip the test if none of the autoscaler pods are running.
+puts "Autoscaler inactive." if have == 0
+exit_skipping_test if have == 0
+
+# Fail the test if some, but not all of the autoscaler pods are
+# running.
+fail "Autoscaler only partially active." if have < STATEFULSETS.length
```

Comment:
Maybe add `x out of y running`. Or even printing the one that didn't come online.

Reply:
Hm. `x out of y` is trivial. The list we can see after the test run via `kubectl get pods ...` ... Or add the output to the checking loop.

Reply:
Done in commit a36b11ea4c26c48873d3061aea8cd416486b68c7